appveyor.yml

```diff
@@ -39,6 +39,12 @@ environment:
       PMDARIMA_MPL_DEBUG: "true"
       PYTHON: "C:\\Python36-x64"
 
+    - PYTHON_ARCH: "64"
+      PYTHON_VERSION: "3.7.1"
+      MPL_VERSION: "2.2.3"
+      PMDARIMA_MPL_DEBUG: "true"
+      PYTHON: "C:\\Python36-x64"
```

Comment:
I think this should be `"C:\\Python37-x64"`

Reply:
Good catch! Resolved